status/profile_plots.py

```diff
@@ -227,13 +198,13 @@ def main(args):
             else:
                 if args.deployment:
                     continue
-            print deployment['deployment_dir']
+            print(deployment['deployment_dir'])
             path = os.path.join(args.path, deployment['deployment_dir'])
             if not os.path.exists(path):
                 os.makedirs(path)
             plot_deployment(deployment, path)
         except Exception:
-            print "Error processing"
+            print("Error processing")
```

Comment:
Should move these to a proper logging framework.  We don't have to do this in this commit since it's legacy code, but it should be put in the issues if we decide not to implement this right now.  Also this currently is printing an error to stdout, which would lead to stderr not working as expected.  At the very least, this should be something like `print("Error processing", file=sys.stderr)`, since this is an error condition.